Quadriga/src/main/java/edu/asu/spring/quadriga/authentication/UserService.java

```diff
@@ -14,20 +16,23 @@
 import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
 import edu.asu.spring.quadriga.service.IUserManager;
 import edu.asu.spring.quadriga.service.QuadrigaUserDetails;
+import edu.asu.spring.quadriga.web.LoginController;
 import edu.asu.spring.quadriga.web.login.QuadrigaGrantedAuthority;
 
 @Service("userService")
 public class UserService implements UserDetailsService {
 
 	@Autowired
 	private IUserManager userManager;
-	
+	private static final Logger logger = LoggerFactory.getLogger(UserService.class);
```

Comment:
Agreed that the logger need not be static. I observed that almost every class in Quadriga that uses a logger has declared logger as static. Should I make changes in all the relevant classes?